tests.py

```diff
@@ -9,6 +9,55 @@
 import tempfile
 import unittest
 
+from objgraph import obj_node_id
+from objgraph import show_graph
+
+try:
+  from cStringIO import StringIO
+except ImportError:
+  from io import StringIO
+
+
+class Python25CompatibleTestCaseMixin:
+
+    def assertRegexpMatches(self, text, expected_regexp, msg=None):
+        if isinstance(expected_regexp, basestring):
+            expected_regexp = re.compile(expected_regexp)
+        if not expected_regexp.search(text):
+            msg = msg or "Regexp didn't match"
+            msg = '%s: %r not found in %r' % (msg, expected_regexp.pattern, text)
+            raise self.failureException(msg)
+
+    def assertIsNotNone(self, value):
+        self.assertTrue(value is not None)
+
+
+# Unit tests
+
+
+def empty_edge_function(obj):
+  return []
+
+
+class TestObject:
+  pass
+
+
+class ShowGraphTest(unittest.TestCase, Python25CompatibleTestCaseMixin):
+  """Tests for the show_graph function."""
+
+  def test_basic_file_output(self):
+    obj = TestObject()
+    output = StringIO()
+    show_graph([obj], empty_edge_function, False, output=output)
+    output_value = output.getvalue()
+    self.assertIsNotNone(output_value)
```

Comment:
I can't imagine a situation where StringIO().getvalue() would return None.  It might return an empty string, if nothing was written in it, but None?

This assertion dosen't seem to be useful, and so can be removed -- this would also make Python25CompatibleTestCaseMixin a bit smaller.